browser/ui/webui/speedreader/speedreader_toolbar_ui.cc

```diff
@@ -41,8 +43,10 @@ SpeedreaderToolbarUI::SpeedreaderToolbarUI(content::WebUI* web_ui,
     source->AddString(str.name, l10n_str);
   }
 
+#if BUILDFLAG(ENABLE_AI_CHAT)
   source->AddBoolean("aiChatFeatureEnabled",
```

Comment:
What's the value of this boolean when build flag is disabled? Does it return undefined on the source side?

Reply:
here's where the build flag is set:
https://github.com/brave/brave-core/blob/1b28357c8c81fad07ab37cf387113a138bba4b4a/components/ai_chat/common/buildflags/buildflags.gni#L10-L13

Our build system (through `build/build_config.h`) can check for these GN defines